bridge_ui/src/hooks/useHandleTransfer.tsx

```diff
@@ -393,7 +395,16 @@ async function alephium(
         return result.txId
       }
     )
+    await TransactionDB.getInstance().txs.put(new Transaction(
+      txInfo.txId,
+      signer.account.address,
+      CHAIN_ID_ALEPHIUM,
+      targetChain,
+      txInfo.sequence,
+      "Pending"
```

Comment:
Do we also want to cache information of the transfer tx, like amount, toAddress, .etc?

Reply:
I'm hesitant about this. 

If we save the tx info just for redemption, I guess these fields are not necessary because the UI will show these fields when the user redeems(parse from the VAA). 

But if we want users to be able to find all transaction information more easily, then it would be great to cache these fields.